### DISCUSSION TITLE
cant update firefly

### DISCUSSION THREAD

**Discussion by Jmoney6** [iotaledger/firefly#2292] [Help & Support] [answered]
this is the message i got and what steps do i need to do to fix, thanks
App Version: 1.3.2

platform: macOS
platformVersion: 10.15.7
platformArchitecture: x64
cpuCount: 4
totalMem: 8192.0 MB
freeMem: 424.7 MB

Error Type: Main Unhandled Promise Rejection

Error: Uncaught Error: Cannot update while running on a read-only volume. The application is on a read-only volume. Please move the application and try again. If you're on macOS Sierra or later, you'll need to move the application out of the Downloads directory. See https://github.com/Squirrel/Squirrel.Mac/issues/182 for more information.

**Comment by MiKeF05** [comment, marked as answer]
You can try to download Firefly directly from firefly.iota.org and install it.
